src/test/java/org/newrelic/jmx/JMXFetcherTest.java

```diff
@@ -73,6 +75,7 @@ public void testJMXWithSSL() throws Exception {
 
 
     public void testJMXFetching(JMXFetcher jmxFetcher) throws Exception {
+        Logging.setup(Logger.getLogger("nrjmx"), true);
```

Comment:
why?

Reply:
Because I noticed that the tests were failing silently in the jenkins machines. This way we can get more insights in the jenkins task log.

Reply:
great, thx!